account_asset_management/models/account_asset_profile.py

```diff
@@ -18,31 +18,24 @@ class AccountAssetProfile(models.Model):
         string='Analytic account')
     account_asset_id = fields.Many2one(
         comodel_name='account.account',
-        domain=[('deprecated', '=', False)],
```

Comment:
Put the domain here, but with quotes:

`domain="[('company_id', '=', company_id), ('type', '=', 'general')]"`